app/views/pages/home.html.erb

```diff
@@ -1,2 +1,11 @@
-<center><h1>Welcome to Vesta</h1></center>
+<div class="homepage-hero">
+  <h1>Welcome to Vesta</h1>
+  <p>
+  Vesta is a web app that is used to manage housing assignments and draws, nominally in a university setting.
```

Comment:
"Vesta is a web app that manages the housing process for Yale's residential colleges." is probably better for pilot